apps/web-giddh/src/app/theme/tax-control/tax-control.component.ts

```diff
@@ -143,10 +151,14 @@ export class TaxControlComponent implements OnInit, OnDestroy, OnChanges {
                         }
                     }
                 } else {
-                    taxObj.amount = tx.taxDetail[0].taxValue;
+                    taxObj.amount = tax.taxDetail[0].taxValue;
+                }
+                if (this.customerCountryName !== 'India' && tax.taxType === 'gst') {
```

Comment:
User can apply CESS, InputGST, etc. also on Sales which have different taxType values than 'gst' please check and confirm in that case these taxes will get checked by default

Reply:
Yes prateek have asked on card and implement on ledger too.

Reply:
User can intentionally add gst type taxes but have removed default selection